espflash/src/cli/mod.rs

```diff
@@ -972,6 +986,31 @@ pub fn make_flash_data(
     )
 }
 
+pub fn write_bin(args: WriteBinArgs, config: &Config) -> Result<()> {
```

Comment:
We should add a docs here as well.

Reply:
Why? The `cli` module will not be included in the documentation moving forward.